pkgs/development/compilers/go/1.11.nix

```diff
@@ -178,6 +178,7 @@ stdenv.mkDerivation rec {
 
   preFixup = ''
     rm -r $out/share/go/pkg/bootstrap
+    rm -r $out/share/go/pkg/obj
```

Comment:
But does not then everybody has to recompile the standardlibary on there own?

Reply:
So I'm pretty sure this is the object cache directory - the actual libraries end up in folders called $GOOS_$GOARCH and a tool and include folder. It's also not shipped by go in their download (https://golang.org/dl/). Oddly enough it also doesn't seem to be working since the folder is still present.

Reply:
Nevermind - I was confused. The obj folder is correctly deleted and running go install with the modified package doesn't appear to cause any standard library recompilation.

Reply:
For reference - the go release tool also deletes this directory.

https://go-review.googlesource.com/c/build/+/82095

Reply:
I mixed up the prefix, because the static archives where right below this.